<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsDialog</class>
 <widget class="QDialog" name="settingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="main">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="dFile_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>94</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Default File</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dFile_lineEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;None&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dFile_pushButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="openLastAtStart_checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>230</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Open Last Opened File At Program Startup</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="startup_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>46</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Startup</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="openDefaultAtStart_checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>248</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Open Selected Default File At Program Startup</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QFrame" name="OK_Cancel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>180</width>
        <height>45</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="okay_pushButton">
         <property name="text">
          <string>Okay</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancel_pushButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
